be/src/olap/schema_change.h

```diff
@@ -268,7 +268,7 @@ class SchemaChangeHandler {
     OLAPStatus process_alter_tablet(AlterTabletType alter_tablet_type,
                                    const TAlterTabletReq& request);
 
-    OLAPStatus schema_version_convert(TabletSharedPtr ref_tablet,
+    OLAPStatus schema_version_convert(TabletSharedPtr base_tablet,
```

Comment:
call this function in delta_writer

Reply:
I will change this name in a following commit. So call it in the next commit.